generators/php.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * @license
+ * Visual Blocks Language
+ *
+ * Copyright 2012 Google Inc.
```

Comment:
New files should be dated with the current year 2015.  [Change here and all other new files.]

You could also change Google Inc to be yourself or your company, but that would mean our lawyers would need to contact you with stuff to sign.  Thanks for leaving this as Google, with yourself as the author below.  [No change needed.]